en/docs/setup/migrating-to-590.md

```diff
@@ -118,14 +115,17 @@ Follow the steps below to perform the upgrade.
     `           <NEW_IS_HOME>/repository/deployment/server/userstores          `
     directory.
     
-    !!! warning
-        If you are using a version prior to WSO2 Identity Server 5.8.0, then proceed to the steps 6 to 9 
-        otherwise directly proceed to step 9.
-    
-6.  Do the following database updates:  
-    1.  Download the [migration resources](../../assets/attachments/migration/wso2is-5.9.0-migration.zip)
+6. Make sure that you have migrated the configurations into NEW version
+   as advised in
+   [preparing for migration section.](../../setup/migrating-preparing-for-migration/#migrating-the-configurations)
```

Comment:
Ensure that you have migrated the configurations over to the NEW version as instructed in the ....

Reply:
fixed